concepts/objects/about.md

```diff
@@ -0,0 +1,266 @@
+# About
+
+## Explanation
+
+Besides primitive data types like `number` and `string`, there is another important data type in JavaScript called `object`.
+Objects are collections of key-value pairs.
+As such, they can be used as what is referred to as maps or dictionaries in other languages.
+
+In other languages, all values in a map often need to have the same data type.
+In JavaScript, only the type of the key is restricted: it has to be a string.
+The values inside one object can have different types.
+They can be primitive types like numbers but also arrays, other objects or even functions.
+This makes objects very versatile so that they are also key entities for [object-oriented programming][oop] (OOP) in JavaScript.
+
+In the following we will focus on objects as collections or maps. Other use cases of objects will be covered in other concepts, see e.g., [Classes][concept-classes].
```

Comment:
```suggestion
In the following we will focus on objects as collections or maps. Other use cases of objects will be covered in other concepts, see e.g. [Classes][concept-classes].
```

Reply:
@SaschaMann Are you sure about this? I read [here](https://exercism.lol/docs/building/markdown/style-guide) that we use American English and from what read there you always put in the comma. 🤔

Reply:
I have never seen a comma after `e.g.`, hence my suggestion, but apparently you're right and American English puts a comma there indeed. The doc you linked to gets that wrong too funnily enough.

The more you know... :D